customize.dist/translations/messages.de.js

```diff
@@ -0,0 +1,230 @@
+
+      out.poll_p_save = "Deine Einstellungen werden sofort automatisch gesichtert.";
+      out.poll_p_encryption = "Alle Eingaben sind verschlüsselt, deshalb haben nur Leute im Besitz des Links Zugriff. Selbst der Server sieht nicht was du änderst.";
+      out.poll_p_howtouse = "Gib deinen Namen in das Eingabefeld unten ein und wähle die Felder für die Zeiten aus, zu welchen verfügbar bist.";
+
+      out.promptName = "Wie heißt du?";
+
+      out.wizardButton = 'Assistent';
+      out.wizardLog = "Klicke auf den Button links oben um zur Umfrage zurückzukehren.";
+      out.wizardTitle = "Nutze den Assistenten um deine Umfrage zu erstellen.";
+      out.wizardConfirm = "Bist du wirklich bereit die angegebenen Optionen bereits zu deiner Umfrage hinzuzufügen?";
+
+      out.poll_closeWizardButton = "Assistenten schließen";
+      out.poll_closeWizardButtonTitle = "Assistenten schließen";
+      out.poll_wizardComputeButton = "Optionen übernehmen";
+      out.poll_wizardClearButton = "Tabelle leeren";
+      out.poll_wizardDescription = "Erstellt automatisch die Optionen indem eine beliebige Anzahl von Daten und Zeiten eingegeben wird.";
+      out.poll_wizardAddDateButton = "+ Daten";
+      out.poll_wizardAddTimeButton = "+ Zeiten";
+
+      out.poll_addUserButton = "+ Nutzer*innen";
+      out.poll_addUserButtonTitle = "Zum Hinzufügen einer Nutzer*in klicken";
+      out.poll_addOptionButton = "+ Optionen";
+      out.poll_addOptionButtonTitle = "Zum Hinzufügen einer Option klicken";
+      out.poll_addOption = "Eine Option vorschlagen";
+      out.poll_optionPlaceholder = "Option";
+      out.poll_addUser = "Einen Namen eingeben";
+      out.poll_userPlaceholder = "Dein Name";
+      out.poll_removeOption = "Bist du sicher, dass du diese Option entfernen möchtest?";
+      out.poll_removeOptionTitle = "Zeile entfernen";
+      out.poll_removeUser = "Bist du sicher, dass du diese Nutzer*in entfernen möchtest?";
+      out.poll_removeUserTitle = "Spalte entfernen";
+      out.poll_editOption = "Bist du sicher, dass du diese Option bearbeiten möchtest?";
+      out.poll_editOptionTitle = "Zeile bearbeiten";
+      out.poll_editUser = "Bist du sicher, dass du diese Nutzer*in bearbeiten möchtest?";
+      out.poll_editUserTitle = "Spalte bearbeiten";
+
+      out.poll_titleHint = "Titel";
+      out.poll_descriptionHint = "Beschreibung";
+
+      // index.html
+
+      out.main_p1 = 'CryptPad ist ein <strong>kenntnisfreier</strong>, kollaborativer Texteditor, der in Echtzeit arbeitet. Durch Verschlüsselung, die in deinem Browser geschieht, werden deine Daten vorm Server, der Cloud und der NSA geschützt. Der geheime Schlüssel ist im sogenannten <a href="https://de.wikipedia.org/wiki/Fragmentbezeichner">Fragmentbezeichner</a> der URL gespeichert. Dieser ist dem Server nicht zugänglich, wohl aber deinem Browser und damit javascript. Durch Teilen der URL autorisierst du andere und ermöglichst ihnen die Mitarbeit.';
```

Comment:
I would agree with @kpcyrd after giving it a second thought. Therefore it might be better to change every occurrence to *zero knowledge*